ui/DesignSystem/Component/Transaction/Row.svelte

```diff
@@ -2,6 +2,7 @@
   import { Flex } from "../../Primitive";
 
   export let style = null;
+  export let dataCy = null;
```

Comment:
This is for cypress, right? If so it would be good flag that explicitely to avoid confusion with other component attributes.

Reply:
Yes, it is for cypress purposes. But we have a those in all tested components, so I would be hesitant to repeat a comment in every place.